website/snippets/_enterprise-permissions-table.md

```diff
@@ -35,9 +35,9 @@ Account roles enable you to manage the dbt Cloud account and manage the account
 |Project-level permission | Account Admin | Billing admin | Project creator | Security admin | Viewer | 
```

Comment:
is the 'project-level permission' column meant to reference dbt cloud UI pages?

Reply:
oh sorry @gcs272 , i meant to ask if the items under the 'project level permissions' column are meant to specifically map to dbt cloud UI pages? e..g connections is the environment data platform connections page, credentials is the env credentials page, etc.)

![Screenshot 2024-07-09 at 09 29 34](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/77c5d107-7d17-456c-8c91-c10d44e0b8d8)

Reply:
ah, I don't think they're really supposed to be 1:1, but it's pretty close